lib/pavilion/sys_vars/cpu_count.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+from .base_classes import SystemPlugin
+
+
+class CPUCount(SystemPlugin):
+
+    def __init__(self):
+        super().__init__(
+            name='cpu_count',
```

Comment:
I looked at the existing variable names. `host_cpus` seems to be the most appropriate name.